skops/io/_scikeras.py

```diff
@@ -0,0 +1,61 @@
+import io
+import os
+import tempfile
+from typing import Sequence, Type
+
+import tensorflow as tf
+from scikeras.wrappers import KerasClassifier, KerasRegressor
+
+from ._audit import Node
+from ._protocol import PROTOCOL
+from ._utils import Any, LoadContext, SaveContext, get_module
+
+
+def scikeras_get_state(obj: Any, save_context: SaveContext) -> dict[str, Any]:
+    res = {
+        "__class__": obj.__class__.__name__,
+        "__module__": get_module(type(obj)),
+        "__loader__": "SciKerasNode",
+    }
+
+    obj_id = save_context.memoize(obj)
+    f_name = f"{obj_id}.keras"
+
+    with tempfile.TemporaryDirectory() as temp_dir:
+        file_name = os.path.join(temp_dir, "model.keras")
+        obj.model.save(file_name)
+        save_context.zip_file.write(file_name, f_name)
+
+    res.update(type="scikeras", file=f_name)
+    return res
+
+
+class SciKerasNode(Node):
+    def __init__(
+        self,
+        state: dict[str, Any],
+        load_context: LoadContext,
+        trusted: bool | Sequence[str] = False,
+    ) -> None:
+        super().__init__(state, load_context, trusted)
+        self.trusted = self._get_trusted(trusted, [KerasClassifier, KerasRegressor])
```

Comment:
we shouldn't be trusting any of keras objects by default for now.

Reply:
Fixed 👍🏽